protocol/agents.json

```diff
@@ -208,11 +203,10 @@
     "cocoa.macos": "ably-cocoa/${version} macOS/0.0.0",
     "cocoa.tvos": "ably-cocoa/${version} tvOS/0.0.0",
     "dotnet": "ably-dotnet/${version}",
-    "dotnet.framework": "ably-dotnet/${version} dotnet-framework/0.0.0",
-    "dotnet.uwp": "ably-dotnet/${version} uwp",
```

Comment:
Are you sure this was never used in any previously released SDK?

Reply:
We have removed support for this since ably-dotnet 1.2.7 (https://github.com/ably/ably-dotnet/pull/1101).  As far as agent headers are concerned, there's no point in having the mapping since we are not emitting `uwp` as a part of agent-headers ( it doesn't exist in the first place).

Reply:
> there's no point in having the mapping since we are not emitting uwp as a part of agent-headers ( it doesn't exist in the first place).

Ok thanks, I just wanted to clarify that 👍